instat/static/InstatObject/R/Backend_Components/summary_functions.R

```diff
@@ -1456,8 +1456,10 @@ DataBook$set("public", "summary_table", function(data_name, columns_to_summarise
     }
   }
   shaped_cell_values <- shaped_cell_values %>% dplyr::mutate(value = round(value, signif_fig))
-if (is.null(shaped_cell_values$`summary-variable`)){
-  shaped_cell_values <- shaped_cell_values %>% dplyr::mutate(summary = as.factor(summary)) %>% dplyr::mutate(summary = forcats::fct_relevel(summary, summaries_display))
+if (treat_columns_as_factor){
+  shaped_cell_values <- shaped_cell_values %>%
+  dplyr::mutate(summary = as.factor(summary)) %>% dplyr::mutate(summary = forcats::fct_relevel(summary, summaries_display)) %>%
+  dplyr::mutate(variable = as.factor(variable)) %>% dplyr::mutate(variable= forcats::fct_relevel(variable, columns_to_summarise))
```

Comment:
Do these 3 lines form a single statement? If so, would we normally use an indent to show this?
```suggestion
  shaped_cell_values <- shaped_cell_values %>%
      dplyr::mutate(summary = as.factor(summary)) %>% dplyr::mutate(summary = forcats::fct_relevel(summary, summaries_display)) %>%
      dplyr::mutate(variable = as.factor(variable)) %>% dplyr::mutate(variable= forcats::fct_relevel(variable, columns_to_summarise))
```
(I'm not an R programmer so if my suggestion is an ignorant one, then please forgive me!)